src/test/java/plugins/WarningsPluginTest.java

```diff
@@ -111,6 +112,15 @@ public void configure(WarningsBuildSettings settings) {
             assertThat(driver, hasContent(title + ": " + warningsPerAxis.get(axis.name)));
         }
 
+        // sometimes the details is not refreshed yet (https://cloudbees.atlassian.net/browse/CJP-3668) 
```

Comment:
An internal issue tracking number that has no usage in OSS.
Maybe replace it with the link to JENKINS-31431?